src/Local/Microsoft.MSBuildCache.Local.csproj

```diff
@@ -37,4 +38,5 @@
     <FilesToSign Include="$(TargetPath)" Authenticode="Microsoft400" StrongName="StrongName" />
   </ItemGroup>
   <Import Project="$(RepoRoot)build\MSBuildExtensionPackage.targets" />
+  <Import Project="..\ILRepack.targets" />
```

Comment:
Is there any case where this isn't imported alongside `MSBuildExtensionPackage.targets`? If not, and if you combine this file and the props file, then might as well just import this in `MSBuildExtensionPackage.targets`

Reply:
`ILRepack.targets` isn't even that large so I wouldn't even be opposed to just inlining it into `MSBuildExtensionPackage.targets`